templates/institutional-landing/index.json-ld

```diff
@@ -0,0 +1,179 @@
+				"path": "reports/a11y-1-fr.html"
+			},
+			{
+				"title": "Canada.ca theme - A reference implementation of the Canada.ca Content and Information Architecture Specification, the Canada.ca Content Style Guide and the Canada.ca Design System",
+				"language": "en",
+				"url": "https://wet-boew.github.io/GCWeb/templates/institutional-landing/institutional-landing-en.html",
+				"path": "reports/a11y-2-en.html"
+			},
+			{
+				"title": "Thème Canada.ca - Une implémentation de référence de la spécification de contenu et d'architecture de l'information de Canada.ca, du guide de style de contenu de Canada.ca et du système de conception de Canada.ca",
+				"language": "fr",
+				"url": "https://wet-boew.github.io/GCWeb/templates/institutional-landing/institutional-landing-fr.html",
+				"path": "reports/a11y-2-fr.html"
+			}
+		]
+	},
+	"dependencies": [
+		"Intro block",
+		"Featured link",
+		"Most requested",
+		"Services and information",
+		"Social media channels",
+		"Minister or institutional head"
+	],
+	"a11yGuidance": "No accessibility guidance.",
+	"variations": [
+		{
+			"name": {
+				"en": "Institutional landing page",
+				"fr": "Pages d'accueil institutionnelle"
+			},
+			"status": "stable",
+			"description": {
+				"en": "Use this template as the landing page for Government of Canada institutions and organizations.",
+				"fr": "Utilisez ce modèle comme page d'accueil pour les institutions et organismes du gouvernement du Canada."
+			},
+			"guidance": {
+				"en": "https://design.canada.ca/mandatory-templates/institutional-profile-pages.html",
+				"fr": "https://conception.canada.ca/modeles-obligatoire/pages-profil-institutionnel.html"
+			},
+			"iteration": "_:iteration_ilp_2",
+			"example": [
+				{
+					"en": { "href": "institutional-landing-en.html", "text": "Institutional landing page" },
+					"fr": { "href": "institutional-landing-fr.html", "text": "Pages d'accueil institutionnelle" }
+				}
+			],
+			"implementation": [
+				"_:implement_ilp"
+			],
+			"history": [
+				{
+					"en": "March 2024 - Stabilization of the page template.",
+					"fr": "Mars 2024 - Stabilisation du gabarit de page."
+				},
+				{
+					"en": "June 2021 - Provisional implementation of the page template.",
+					"fr": "Juin 2021 - Implémentation provisoire du gabarit de page."
+				}
+			]
+		}
+	],
+	"implementation": [
+		{
+			"@id": "_:implement_ilp",
+			"iteration": "_:iteration_ilp_1",
+			"name": {
+				"en": "Standard",
+				"fr": "Standard"
+			},
+			"introduction": {
+				"en": "This implementation is meant for developers/publishers adding the component manually.",
+				"fr": "Cette implémentation est destinée aux développeurs/éditeurs qui ajoutent le composant manuellement."
+			},
+			"instructions": {
+				"en": [
+					"Refer to the working example for a code sample.",
+					"Refer to the official guidance for more information on how to implement this page template."
+				],
```

Comment:
Add the following similar instruction:
* The author need to add the `page-type-ilp` CSS class on their page `<body>` element. Note: If you don't, the heading level 2 are not going to be at the right size according to the guidance. We did implemented a temporary workaround to give the time to Web CMS to implement it. The workaround are going to be removed at the same time of the demote/deprecated ILP version 1

Also, a little addition for the last instruction item:
```suggestion
				"en": [
					"Refer to the working example for a code sample.",
					"Refer to the working example and the guidance for more information on how to implement this page template."
				],
```